theme/static/style/style.css

```diff
@@ -325,52 +323,90 @@ pre code {
 
 /* Footer Section */
 
-.footer{
+.footer {
     padding: 2% 0;
     display: inline-block;
     font-size: 1.2rem;
 }
 
-.footer-logo{
+.footer-logo {
     float: right;
     margin-right: 1rem;
     vertical-align: middle;
     overflow-wrap: break-word;
 }
 
-.footer-copyright{
+.footer-copyright {
     line-height: 1.8em;
     vertical-align: middle;
     overflow-wrap: break-word;
 }
 
-.footer-copyright a{
+.footer-copyright a {
     text-decoration: underline;
 }
 
-.footer-sociallinks{
+.footer-sociallinks {
     list-style-type: none;
     text-align: right;
 }
 
-.footer-sociallinks li{
+.footer-sociallinks li {
     display: inline-block;
     margin-right: 1.5rem;
 }
 
-.footer-sociallinks a{
+.footer-sociallinks a {
     font-size: 3.5rem;
 }
 
+/* Features Page */
+
+.horizontally-stacked-list {
+    display: flex;
+    float: left;
+}
+
+.left-floating {
+    float: left;
+}
+
+.testplan {
+    float: right;
+    clear: right;
+    margin-bottom: 5rem;
+}
+
+.testplan-img {
+    margin-bottom: 5rem;
+}
+
+.mark-and-status-bugs {
+    float: left;
+    margin-bottom: 5rem;
+}
+
+.call-to-action-field {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    text-align: center;
+}
+
+.about-panel {
+    padding-top: 2rem;
+}
+
 @media screen and (max-width: 1360px){
```

Comment:
When the window width is shrinked enough for the styles in this query to apply, the images go off-center and stick to the left side of the screen, which does not fit with the overall style of the page. This can be fixed by adding the following properties inside of the query:

```
.col-8 {
    width: 97%;
}

.margin0 {
    margin-left: 7.5%;
    width: 87%;
}
```

`.margin0` refers to the images.

Or try finding another solution if this one doesn't suit you.